### PR TITLE
imac of my dreams

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://junglq.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://junglq.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (AlinaMatuschak) [inline src/index.html:6]: Fix code style for all code

```suggestion
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
+  </head>
+  <body>
+    <div class="wrap">
+      <div class="card" data-qa="card">
+        <img class="card__img" src="images/imac.jpeg"
+          alt="imac">
+
+        <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+        <div class="card__code">Код товара: 195434</div>
```
Comment (AlinaMatuschak) [inline src/index.html:19]: use tags by functionality. The div tag is used as a container for HTML elements. Use one of text tags (p, span)

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  background: url(images/star-active.svg) no-repeat;
+}
+
+.rating__star:nth-child(3) {
+  background: url(images/star-active.svg) no-repeat;
+}
+
+.rating__star:nth-child(4) {
+  background: url(images/star-active.svg) no-repeat;
+}
```
Comment (AlinaMatuschak) [inline src/style.css:91]:
```suggestion
.rating__star:nth-child(1),
.rating__star:nth-child(2),
.rating__star:nth-child(3),
.rating__star:nth-child(4) {
  background: url(images/star-active.svg) no-repeat;
}
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  align-items: center;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  color: #fff;
+  font-weight: 700;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  transition: all 0.3s ease 0s;
```
Comment (AlinaMatuschak) [inline src/style.css:137]: bad practice to add a transition for `all` styles. Replace `all` with a style name that will change property values smoothly

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,43 @@
...
+
+        <div class="card__rating rating">
+          <div class="rating__stars">
```
Comment (AlinaMatuschak) [inline src/index.html:22]: add rating__stars--4 class. If you change rating__stars--4 modifier to rating__stars--2 or other - number of selected stars should change correctly.

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Really good work 🚀. Just fix small issues:

Look 10 point on [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,50 @@
...
+      <div class="card" data-qa="card">
+        <img class="card__img" src="images/imac.jpeg"
+          alt="imac">
+
+        <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+        <p class="card__code">Код товара: 195434</p>
+
+        <div class="card__rating rating">
+          <div class="rating__stars rating__stars--3">
```
Comment (AlinaMatuschak) [inline src/index.html:28]: On mockup 4 stars. I left previous feedback for adding all styles for this element and keep it operational.

Are you sure that you redeployed your project? I don't see new styles on the browser

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,50 @@
...
+
+        <div class="card__price price">
+          <div class="price__item">Цена:</div>
+          <div class="price__uah">69 999 грн</div>
+        </div>
+
+        <a href="#" class="card__button"
+          data-qa="hover">
+            Купить
+        </a>
```
Comment (AlinaMatuschak) [inline src/index.html:46]: I see more than one mistake like this

```suggestion
        <a
          href="#"
          class="card__button"
          data-qa="hover"
        >
            Купить
        </a>
```

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Fix 10 point at [checlist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md). If you don't understand what this means, write to the slack channel (on the next task you has the same mistake)

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,24 @@
+@import "./reset.css";
+
+html,
+body {
+  height: 100%;
+  background-color: #ff6347;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
```
Comment (aionova) [inline src/style.css:8]: it's better to use variables for colors

File: src/reset.css [hunk 9]
```diff
@@ -0,0 +1,7 @@
+*,
+*:after,
+*:before {
+  margin: 0;
+	padding: 0;
+  box-sizing: border-box;
+}
```
Comment (aionova) [inline src/reset.css:7]: use * selector only for box-sizing property. don't use it for margins and paddings

Comment (aionova) [reply src/reset.css:7]: add empty line at the end of the file

File: src/blocks/card.css [hunk 10]
```diff
@@ -0,0 +1,60 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
```
Comment (aionova) [inline src/blocks/card.css:4]: don't set width for BEM-block

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 20px;
+}
+
+@import "./blocks/card.css";
+@import "./blocks/rating.css";
+@import "./blocks/price.css";
+
```
Comment (aionova) [inline src/style.css:22]: imports are usually written in the beginning of the file

**Review by aionova — APPROVED** [review]
GJ!

**Pull request closed without merging**